### PR TITLE
v4.3.1 - Alt+Rightclick Move

### PR DESCRIPTION
**Pull request by MrxSiN**
Alt+Rightclick forces your hero to move in that Direction rather than the regular Rightclick which moves your hero to that point.

Example: http://www.twitch.tv/sing_sing/v/5094104?t=2h48m43s Sing trying to leap over trees using normal Rightclick movement. However, if he used Alt+Rightclick anywhere on this red line: http://i.imgur.com/Qbjs1ZU.jpg: he would be facing the appropriate direction for a safe leap.
TL;DR - Alt+Rightclick movement is incredibly useful to face proper directions in narrow positions, especially with Mirana's Leap, Shadowfriend's Razes, and Forcestaves.

Must be turned on through console
dota_unit_allow_moveto_direction 1

By this config "Alt" is the "`"

From Dota 2 Reddit

### CODE REVIEW

**Comment by loopuleasa** [comment]
It is implemented like that now.
